tests/csapi/apidoc_room_create_test.go

```diff
@@ -61,6 +62,78 @@ func TestRoomCreate(t *testing.T) {
 			})
 			content := alice.MustGetStateEventContent(t, roomID, "m.room.topic", "")
 			must.MatchGJSON(t, content, match.JSONKeyEqual("topic", "Test Room"))
+
+			// Rich topics not implemented yet on Dendrite
+			runtime.SkipIf(t, runtime.Dendrite)
+
+			// The plain text topic is duplicated into m.topic
+			must.MatchGJSON(t, content,
+				match.JSONKeyArrayOfSize("m\\.topic.m\\.text", 1),
+				match.JSONKeyPresent("m\\.topic.m\\.text.0.body"),
+				match.JSONKeyEqual("m\\.topic.m\\.text.0.body", "Test Room"))
+
+			// The mime type must be unset or text/plain
+			mime := content.Get("m\\.topic.m\\.text.0.mimetype")
+			if mime.Exists() {
+				must.Equal(t, mime.String(), "text/plain", "no m.topic content block")
+			}
+		})
```

Comment:
We still want the first part of the test to run and succeed/fail on Dendrite so I think we still need this bottom part to be separate.